### PR TITLE
Sync CHANGELOG with Calyptia-Fluentd v1.4.2 release

### PR DESCRIPTION
**Pull request by cosmo0920**
Signed-off-by: Hiroshi Hatake <hatake@calyptia.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,935 @@
+# Calyptia Fluentd 1 changelog
+
+## Release v1.4.0 - 2022/06/30
+
+### News
+
+* Upgrade Fluentd to v1.15.0
+  * In this version of Fluentd, it starts to handle YAML configuration.
```
Comment (patrick-stephens) [inline CHANGELOG.md:8]:
```suggestion
  * In this version of Fluentd, experimental support for handling YAML configuration is provided.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,935 @@
...
+* Use vanilla fluent-plugin-splunk-hec v1.12.13 which is resloved Ruby 3 compatibility issues
```
Comment (patrick-stephens) [inline CHANGELOG.md:9]:
```suggestion
* Use the vanilla fluent-plugin-splunk-hec v1.12.13 which resolves Ruby 3 compatibility issues
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,935 @@
...
+* fluent-plugin-parser-winevt_xml v0.2.4
+* fluent-plugin-windows-eventlog v0.8.1
+
+## Release v1.3.5 - 2022/02/28
+
+### News
+
+* Update Fluentd version to v1.14.5
+* Including fixes for fluent-plugin-opensearch
+* Update Elasticsearch 8 compatible flunet-plugin-elasticsearch
```
Comment (patrick-stephens) [inline CHANGELOG.md:294]:
```suggestion
* Update to Elasticsearch 8 compatible fluent-plugin-elasticsearch
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,935 @@
...
+* fluent-plugin-parser-winevt_xml v0.2.3
+* fluent-plugin-windows-eventlog v0.8.1
+
+## Release v1.3.1 - 2021/11/01
+
+### News
+
+* Update using Fluentd to v1.14.2 vanilla
+* Update Elasticsearch related gems and plugin
+  * Note that this version blocks for communicating with OpenSearch. We'll support OpenSearch later with another plugin.
```
Comment (patrick-stephens) [inline CHANGELOG.md:564]:
```suggestion
  * Note that this version does not support OpenSearch.
```

File: CHANGELOG.md (hunk 1 above)
Comment (cosmo0920) [reply CHANGELOG.md:8]: We Fluentd maintainer also provides the detail document to use YAML configuration format: 
https://docs.fluentd.org/configuration/config-file-yaml

In Fluentd side, it's not experimental feature.

Comment (patrick-stephens) [reply CHANGELOG.md:8]: Fair enough, maybe:
```suggestion
* This version of Fluentd provides support for YAML configuration as well.
```

Comment (cosmo0920) [reply CHANGELOG.md:8]: Hmm, it's reasonable for me. I'll handle to modify with your suggestion later.

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,1081 @@
+# Calyptia Fluentd 1 changelog
+
+## Release v1.4.2 - 2022/11/29
+
+### News
+
+* Upgrade Rubies to 3.1.3/3.0.5
```
Comment (patrick-stephens) [inline CHANGELOG.md:7]: Should this be `Ruby`?

**Review by patrick-stephens — APPROVED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (cosmo0920) [reply CHANGELOG.md:7]: Not sure but I sometimes see the plural form of Ruby like as this line.

**Comment by cosmo0920** [comment]
@agup006 Could you add +1 this PR? Some of customers want public Changelog for Calyptia Fluentd.

File: CHANGELOG.md (hunk 5 above)
Comment (patrick-stephens) [reply CHANGELOG.md:7]: Yeah it looks wrong - there is only one version on each target so I'd just use the singular.

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**